specification/ai/data-plane/HealthInsights/readme.md

```diff
@@ -4,10 +4,10 @@
 
 Configuration for generating Health Insights SDK.
 
-The current release is `2023-09-01-preview`.
+The current release is `2024-04-01`.
 
 ```yaml
-tag: 2023-09-01-preview
+tag: 2024-04-01
```

Comment:
This is causing some of the swagger failures, you need to revert the change to this line

Reply:
Git diff made this look like it was for the preview spec...disregard the previous comment.